locksmith/src/operations/userOperations.ts

```diff
@@ -135,9 +135,9 @@ namespace UserOperations {
     publicKey: string
   ): Promise<boolean> => {
     const paymentProcessor = new PaymentProcessor(
-      config.stripeSecret,
-      config.web3ProviderHost,
-      config.unlockContractAddress
+      config.stripeSecret!,
```

Comment:
These can be undefined, no real behavior change but since the config file is TS I needed to indicate that we always expect values to exist.